composer.lock

```diff
@@ -1,39 +1,39 @@
 {
     "_readme": [
         "This file locks the dependencies of your project to a known state",
-        "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#composer-lock-the-lock-file",
+        "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#installing-dependencies",
         "This file is @generated automatically"
     ],
-    "content-hash": "bf6a77e5921f7f5808369aba1890007d",
+    "content-hash": "00481d1a24291b1ebea17f475003bf25",
     "packages": [
         {
             "name": "doctrine/instantiator",
-            "version": "1.1.0",
```

Comment:
Note that [`doctrine/instantiator:^1.1.0`](https://github.com/doctrine/instantiator/blob/1.1.0/composer.json#L19) requires PHP 7.1.